src/privatesend-client.h

```diff
@@ -10,17 +10,20 @@
 #include "wallet/wallet.h"
 #include "privatesend-util.h"
 
-class CPrivateSendClient;
+class CPrivateSendClientManager;
 class CConnman;
 
 static const int DENOMS_COUNT_MAX                   = 100;
 
+static const int MIN_PRIVATESEND_SESSIONS           = 1;
 static const int MIN_PRIVATESEND_ROUNDS             = 2;
 static const int MIN_PRIVATESEND_AMOUNT             = 2;
 static const int MIN_PRIVATESEND_LIQUIDITY          = 0;
+static const int MAX_PRIVATESEND_SESSIONS           = 10;
 static const int MAX_PRIVATESEND_ROUNDS             = 16;
 static const int MAX_PRIVATESEND_AMOUNT             = MAX_MONEY / COIN;
 static const int MAX_PRIVATESEND_LIQUIDITY          = 100;
+static const int DEFAULT_PRIVATESEND_SESSIONS       = 4;
```

Comment:
Don't want to keep too many connections open to avoid high bandwidth usage while still having some positive effect of using multiple sessions even with default settings. 4 and 10 seems like reasonable numbers for normal and extreme usage respectively IMO :)

Reply:
Alright, we probably want to put some documentation out about how much bandwidth it'll use, etc on different values and if that might affect privacy in any way. Although that might be more @strophy 's domain?

Reply:
Well, by default node keeps `MAX_OUTBOUND_CONNECTIONS` (8) regular connections open, so each new connections adds some `initial sync` bandwidth (should be negligible) and `relay everything` one which depends on current network activity. Anyway, good that you asked - I realized that we should also bump `MAX_OUTBOUND_MASTERNODE_CONNECTIONS` to make sure mixing sessions fit into the limit, see 213a080 🙂 👍